### ISSUE TITLE
Funcionamiento text-search

### ISSUE THREAD

**Issue by fjcroquevielle** [IIC2413/Syllabus-2020-1#357]
Hola, 
Es solo para estar seguro, cualquier búsqueda text-search que se haga en nuestra API será a travez de Postman o no? Pregunto esto porque a travez de Postman mis consultas funcionan bien,  sin embargo, si pongo como URL localhost:5000/text-search me tira un Internal server error.
Gracias!!

**Comment by nivek0o0** [comment]
Sí. Pero si se te cae porque no llega body está incorrecto, es un caso que probaremos.
